core/templates/domain/topic/creator-topic-summary.model.ts

```diff
@@ -60,7 +64,11 @@ export class CreatorTopicSummary {
       public classroom: string | undefined,
       public thumbnailFilename: string,
       public thumbnailBgColor: string,
-      public urlFragment: string) { }
+      public urlFragment: string,
+      public totalUpcomingChapters: number,
+      public totalOverdueChapters: number,
```

Comment:
Is "total" actually required for these? Also, should it end with "Count" given that the variable represents a number and not a chapter model?

Reply:
Updated.

Reply:
Thanks! Could you also respond to the first question: Is "total" actually required? Note that their backend equivalents do not have the "total".

Reply:
Yes total is required. I have updated the backend equivalents to "total".